.release/buildspec_sign.yml

```diff
@@ -44,6 +44,21 @@ phases:
       - ls -lah $(dirname `head -1 ./$GIT_COMMIT_ID.manifest`)
   post_build:
     commands:
+      - echo "Copying artifacts to generate a latest binary"
```

Comment:
Another thing I just thought of: this means that "latest" in our s3 bucket will be different from the "latest" available through github. Because this runs on every commit it will overwrite the "latest" files in the release bucket every time. 

Also, on failed integ tests, the "latest" artifacts will be marked private and scheduled for deletion. We may wish to update the integ test buildspec to only mark a specific artifact version private, not the whole artifact.